tests/test_backend.py

```diff
@@ -810,6 +812,19 @@ def test_leaky_relu(self):
                 self._run_test_case([_OUTPUT], {_INPUT: x_val})
                 tf.reset_default_graph()
 
+    @skip_caffe2_backend("fails on caffe2 with dim issue")
+    @check_onnxruntime_incompatibility("Mul")
+    @check_tf_max_version("1.5")
+    def test_leaky_relu_old(self):
```

Comment:
on a second thought, how about have ```test_leaky_relu_int``` and ```test_leaky_relu_float```.
```test_leaky_relu_int``` requires tf > 1.5, while ```test_leaky_relu_float``` has no filter?

Reply:
This seems better. I will modify accordingly.